site/src/docs/components/header/code.mdx

```diff
@@ -125,6 +125,8 @@ import { Header, IconSearch } from 'hds-react';
 
 </Playground>
 
+**Note:** Header is designed to be used full width at the top of the page above the main content. Preview here might not scale correctly, <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--with-full-features">view full header example in Storybook</Link>
```

Comment:
Last clause is missing a dot at the end.

Reply:
https://github.com/City-of-Helsinki/helsinki-design-system/pull/1112/commits/9d11b9fb379f70ab235a784619b13308c8ec59bd

Reply:
Excellent! 

PS. We've agreed in HDS that commits should be squashed when fixing/updating previous commits in the same PR. This way history is cleaner and there will be less conflicts later. 

So all three commit could be probably rebased as one as they kinda do the same thing?